tests/unit/test_data_cleaner.py

```diff
@@ -63,4 +65,13 @@ def test_clean_remove_currency_symbols():
 def test_clean_remove_stopwords():
     test = "This is simple text for test!"
     test = clean_remove_stopwords(test)
-    assert test == "This simple text test!"
\ No newline at end of file
+    assert test == "This simple text test!"
+
+
+def test_clean_text_from_specific_characters():
+    characters = ["<", ">"]
+    text = "there is > in < donezia"
+    test = clean_text_from_specific_characters(text, characters)
+
+    assert "<" and ">" not in test
+    assert "there is in donezia" == test
```

Comment:
this is not really the point, but it can be tested as well; 
the trick is in the above assertion ;)